packages/formstr-app/src/components/v1/QuestionCard/InputElements/ShortText.tsx

```diff
@@ -0,0 +1,40 @@
+import { Input } from "antd";
+import QuestionContext from "../../QuestionContext";
+import { AnswerTypes } from "@formstr/sdk/dist/interfaces";
+
+{
+  /* <Input
+  placeholder="Type your answer here"
+  bordered={false}
+  disabled
+  style={{
+    paddingLeft: 0,
+    // borderBottom: "0.5px solid black",
+    borderRadius: 0,
+  }}
+/>; */
+}
+
+interface ShortTextInputSettings {
+  validations: Array<string>;
+}
+interface ShortTextProps {}
+
+const shortText: React.FC<ShortTextProps> = () => {
+  return (
+    <>
+      <Input
+        placeholder="Type your answer here"
```

Comment:
should the placeholder be taken through props.

Can this be used as a building block for question answering as well?